forwarder/parse_commandline_args.py

```diff
@@ -101,6 +101,14 @@ def parse_args():
         type=str,
         env_var="GRAYLOG_LOGGER_ADDRESS",
     )
+    parser.add_argument(
+        "--graphite-server",
+        default="localhost",
+        required=False,
+        help="Graphite metrics server address (Carbon)",
+        # Does it need to be <host>:<port>
+        type=str,
+    )
```

Comment:
It would be nice to be consistent with the equivalent option in the Filewriter:
```
--grafana-carbon-address URI  <host:port> Address to the Grafana (Carbon) metrics service.
```